### PR TITLE
Foundation for using template events

### PR DESCRIPTION
**Pull request by pamil**
| Q               | A
| --------------- | ---
| Bug fix?        | no |
| New feature?    | yes |
| BC breaks?      | no |
| Related tickets | - |
| License         | MIT |

See https://sonata-project.org/bundles/block/master/doc/reference/events.html.

It allows to add a template "@Foo/bar.html.twig" for event "foo.bar" just in four lines of code:

```xml
<service id="xdddd" class="Sylius\Bundle\UiBundle\Block\BlockEventListener">
    <argument>@Foo/bar.html.twig</argument>
    <tag name="kernel.event_listener" event="sonata.block.event.foo.bar" method="onBlockEvent" />
</service>
```

### CODE REVIEW

File: src/Sylius/Bundle/UiBundle/Resources/config/app/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewsk
```
Comment (TheMadeleine) [inline src/Sylius/Bundle/UiBundle/Resources/config/app/config.yml]: :tada:

Comment (pamil) [inline src/Sylius/Bundle/UiBundle/Resources/config/app/config.yml]: 🎉

Comment (pamil) [inline src/Sylius/Bundle/UiBundle/Resources/config/app/config.yml]: FREE COMMIT!

File: app/config/config.yml [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 imports:
     - { resource: "@SyliusCoreBundle/Resources/config/app/config.yml" }
+    - { resource: "@SyliusUiBundle/Resources/config/app/config.yml" }
```
Comment (pjedrzejewski) [inline app/config/config.yml]: Can't we configure it in admin & shop? This is a small but annoying BC break.

Comment (pamil) [inline app/config/config.yml]: Should we duplicate the configuration?

Comment (pjedrzejewski) [inline app/config/config.yml]: I think it would be ok to duplicate it.

**Comment by pjedrzejewski** [comment]
Thanks Kamil!

**Pull request merged**